test.js

```diff
@@ -192,53 +222,69 @@ test('scheduled rewards', async t => {
       }
       assert.deepEqual(log, [
         {
-          address: '0x000000000000000000000000000000000000dEaD',
+          address: '0x000000000000000000000000000000000000dEa2',
           score: '10',
-          scheduledRewards: '4566'
+          scheduledRewardsDelta: '4566'
         },
         {
-          address: '0x000000000000000000000000000000000000dEa2',
+          address: '0x000000000000000000000000000000000000dEa7',
           score: '100',
-          scheduledRewards: '45662'
+          scheduledRewardsDelta: '45662'
         },
         {
-          address: '0x000000000000000000000000000000000000dEaD',
+          address: '0x000000000000000000000000000000000000dEa2',
           score: '10',
-          scheduledRewards: '4566'
+          scheduledRewardsDelta: '4566'
         },
         {
-          address: '0x000000000000000000000000000000000000dEaD',
-          scheduledRewards: '-9132'
+          address: '0x000000000000000000000000000000000000dEa2',
+          scheduledRewardsDelta: '-9132'
         }
       ])
     }
   })
   await t.test('validate signatures', async t => {
```

Comment:
I'm not sure I follow, isn't `t.test()` grouping tests as well?

Reply:
in `node:test`, `describe()` is an alias of `suite()`: https://nodejs.org/api/test.html#describe-and-it-aliases.

I'm not sure when to use `suite()` vs `test()`, do you know how it could improve this test suite?

Reply:
I would use `suite` for grouping related tests and `test` for individual tests, that's how I am using `describe` and `it`.

Basically, only suites are allowed to contain nested things (tests or suites).

```js
suite('scheduled rewards', () => {
  test('empty scheduled rewards', async t => {
    // ...
  })

  test('ignore burner address', async t => {
    // ...
  })

  // etc.
})
```

This way, you also do not need to `await t.test`.

See the example code snippet provided in [Snapshot Testing](https://nodejs.org/api/test.html#snapshot-testing):

```js
// test.js
suite('suite of snapshot tests', () => {
  test('snapshot test', (t) => {
    t.assert.snapshot({ value1: 1, value2: 2 });
    t.assert.snapshot(5);
  });
});
```

Reply:
I understand, thank you, but why is it better?

I assume it's better because you can filter tests better? I never use this functionality so I don't mind. Would you like me to rework this using `suite()`?

Reply:
> Basically, only suites are allowed to contain nested things (tests or suites).

Please see https://nodejs.org/api/test.html#contexttestname-options-fn, node test doesn't have this limitation

Reply:
Switched to suites in
- https://github.com/filecoin-station/spark-rewards/pull/1/commits/84a78543396ffda5cd2dc940aa025a6bcf53362b
- https://github.com/filecoin-station/spark-rewards/pull/1/commits/e2fbde8aceb160778abe2bf403cc318ded4df1ea

Reply:
> I understand, thank you, but why is it better?

I find the test code easier to read this way because it's clear to me which parts define the structure and which parts are actually executed as tests.

> I assume it's better because you can filter tests better?

That's very likely! I don't have any experience with your style of structuring tests, so I cannot tell how it affects test filtering.

> I never use this functionality so I don't mind. Would you like me to rework this using `suite()`?

That would be great, thank you! 🙏🏻 

> Switched to suites

😍